src/Sylius/Bundle/CoreBundle/Migrations/Version20220210135918.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20220210135918 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add guest field to mark orders made by guests';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE sylius_order ADD guest TINYINT(1) DEFAULT \'1\' NOT NULL');
```

Comment:
Won't it break current users carts? Should we at least mention it in upgrade file?

Reply:
yup, added that sql